src/Tests/Service/U2fServiceAuthenticationVerificationTest.php

```diff
@@ -20,17 +20,18 @@
 
 use Mockery as m;
 use PHPUnit_Framework_TestCase as TestCase;
-use Surfnet\StepupU2fBundle\Service\AuthenticationVerificationService;
 use Surfnet\StepupU2fBundle\Service\AuthenticationVerificationResult;
+use Surfnet\StepupU2fBundle\Service\U2fService;
+use Surfnet\StepupU2fBundle\Value\AppId;
 use u2flib_server\Error;
 
-final class AuthenticationVerificationServiceTest extends TestCase
+final class U2fServiceAuthenticationVerificationTest extends TestCase
```

Comment:
Still feel this class tests a lot, perhaps too much. Please document (class docblock) why the tests include expectations on the integration of the U2F lib as we discussed last week.